presto-docs/src/main/sphinx/connector/hudi.rst

```diff
@@ -0,0 +1,92 @@
+====================
+Hudi connector
+====================
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+--------
+Overview
+--------
+
+Apache Hudi is a rich platform to build streaming data lakes with incremental data pipelines
+on a self-managing database layer, while being optimized for lake engines and regular batch
+processing. By adopting Multiversion Concurrency Control (MVCC), where compaction action
+merges logs and base files to produce new file slices and cleaning action gets rid of
+unused/older file slices to reclaim space on the file system, Hudi can support two types of
+tables: Copy-on-Write and Merge-on-Read.
+
+Hudi table state is maintained in commit metadata files, which is essentially a timeline of
+all actions performed on the table at different instants of time that helps provide
+instantaneous views of the table, while also efficiently supporting retrieval of data in
+the order of arrival. To learn more about Hudi, please check out the `overview <https://hudi.apache.org/docs/overview/>`_
+and `quickstart <https://hudi.apache.org/docs/quick-start-guide/>`_
+
+This connector enables querying Hudi tables synced to Hive metastore. The connector uses
+the metastore only to track partition locations. It makes use of the underlying Hudi file
+system and input formats to list data files. Hudi connector runs a partition scanner and
+split generator asynchronously. As the name suggests, the partition scanner will scan all
+the partitions and split generator will list files and generate splits for each pruned
+partition. To learn more about the design of the connector, please check out `RFC-40 <https://github.com/apache/hudi/blob/master/rfc/rfc-44/rfc-44.md>`_.
+
+--------
+Requirements
```

Comment:
IMO, this section is unnecessary. Let's drop it?

Reply:
has modified this section